.travis.yml

```diff
@@ -11,113 +11,11 @@ env:
     - HELM_URL=https://storage.googleapis.com/kubernetes-helm/helm-v2.14.1-linux-amd64.tar.gz
```

Comment:
should we switch here to v3 as well?

Reply:
Not yet, I really don't see the need for building the packaged chart when you just can clone the repo.